coalib/bearlib/aspects/meta.py

```diff
@@ -76,21 +77,41 @@ def __repr__(cls):
         return '<%s %s>' % (type(cls).__name__, repr(cls.__qualname__))
 
 
+def isaspect(item):
+    """
+    This function checks whether or not an object is an instance of
+    ``aspectclass``
+    """
+    if not isinstance(item, (aspectclass, aspectbase)):
+        return False
+
+    return True
+
+
+def assert_aspect(item):
+    """
+    This function raises ``aspectTypeError`` when an object is not an instance
+    of ``aspectclass``
+    """
+    class aspectTypeError(TypeError):
```

Comment:
I am not sure about this but why wouldn't we move the error class into module level? Is there any chance we want to use it outside of `assert_aspect` function?